.circleci/config.yml

```diff
@@ -165,15 +165,15 @@ references:
       - <<: *update_versions
       - deploy:
           name: Install Codacy
-          command: make -e -C .doks/ deploy_to_doks_from_local_dir
+          command: eval make -e -C .doks/ deploy_to_doks_from_local_dir HELM_INSTALL_EXTRA_FLAGS="$EXTRA_FLAGS"
```

Comment:
shouldn't this eval be inside the makefile for the EXTRA_FLAGS?

I wont remember to do eval everywhere